WcaOnRails/.eslintrc.json

```diff
@@ -30,6 +32,7 @@
         "import/no-unresolved": ["error", {
             "ignore": ["semantic-css/"]
         }],
-        "no-console": ["error", { "allow": ["warn", "error"] }]
+        "no-console": ["error", { "allow": ["warn", "error"] }],
+        "jsx-a11y/no-autofocus": "off"
```

Comment:
I frankly don't remember.
I think it might've been something I added because of a line of code I added, then I removed the line of code and forgot to remove this.